iosHackthonApp/iosHackthonApp/Add Meal Screen/AddMealView.swift

```diff
@@ -8,48 +8,125 @@
 
 import SwiftUI
 import shared
+import CoreLocation
 
 struct AddMealView: View {
 
-    @ObservedObject private(set) var viewModel: AddMealViewModel
-
-    var body: some View {
-        ZStack {}
-        .alert(isPresented: $viewModel.showingCollectionCode) {
-            Alert(
-                title: Text(viewModel.code),
-                message: Text(Strings.AddMealScreen.CollectionAlert.message),
-                dismissButton: .default(Text(Strings.Common.ok)))
-        }
-        ZStack {
-            VStack {
-            //TODO: Pass the actual values when UI is complete
-                Button("Add a meal") {
-                    let meal = Meal(
-                        id: "\(viewModel.sdk.getUUID())",
-                        name: "Pizza",
-                        quantity: 2,
-                        availableFromDate: "Tuesday",
-                        expiryDate: "Saturday",
-                        info: "Meat",
-                        hot: false,
-                        locationLat: 51.509865,
-                        locationLong:  -0.118092)
-                    self.viewModel.postMeal(meal: meal)
-                }
-            }
-        }
-        .alert(isPresented: $viewModel.showingError) {
-            Alert(
-                title: Text(Strings.Common.sorry),
-                message: Text(Strings.Common.ErrorAlert.message),
-                dismissButton: .default(Text(Strings.Common.ok)))
-        }
-    }
+	@ObservedObject private(set) var viewModel: AddMealViewModel
+	@State var title: String = ""
+	@State var additionalInfo: String = ""
+	@State var quantity: Int = 0
+	@State var availableFromDate = Date()
+	@State var useByDate = Date()
+	@State var address: String = ""
+	@State var latitude: Float = 0
+	@State var longitude: Float = 0
+	@State var isHot : Bool = true
+
+	var body: some View {
+
+	ZStack {}
+	.alert(isPresented: $viewModel.showingCollectionCode) {
+			Alert(
```

Comment:
Just curious, are we using 4 spaces for a tab or 2 spaces for a tab?

Reply:
Mine was set to 2, but have updated it to 4 to match everyone else.